pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3359,7 +3361,13 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 						Name:          "blue",
 						NetworkSource: v1.NetworkSource{Multus: &virtv1.MultusNetwork{NetworkName: "blue-net"}}}}
 
-					pod = NewPodForVirtualMachine(vmi, k8sv1.PodRunning, testPodNetworkStatus...)
+					pod = NewPodForVirtualMachine(vmi, k8sv1.PodRunning)
```

Comment:
Please introduce `NewPodForVirtualMachineWithMutlusAnnotations` method instead of copy pasting it multiple times.

Reply:
Done